paasta_tools/cli/cmds/local_run.py

```diff
@@ -611,7 +611,7 @@ def paasta_local_run(args):
 
     service = figure_out_service_name(args, soa_dir=args.yelpsoa_config_root)
     cluster = guess_cluster(service=service, args=args)
-    instance = guess_instance(service=service, cluster=cluster, args=args)
+    instance = guess_instance(service=service, args=args)
```

Comment:
I thought I had mentioned that before, but yeah, it was built as a branch of the chronos validation pull request and can't be merged until that one is.